docker-vms/vxfreeswitch/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+EXPOSE 8000
```

Comment:
:+1: on making it the default, these ports are always re-mapped anyway with `docker run -p` for access outside the container.